### PR TITLE
230 exit survey

### PR DESCRIPTION
**Pull request by aphamdd**
Exit survey properly displaying now and removed the extra "open" button. The survey results are now stored in the firebase database.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -57,6 +58,7 @@ function App() {
           <Route exact path="/login" component={Login} />
           <Route exact path="/signup" component={Signup} />
           <Route exact path="/register" component={SignupForm} />
+          <Route exact path="/exit" component={ExitSurveyModal} />
```
Comment (juroc95) [inline src/App.tsx:61]: should it be in app.tsx or dashboard.tsx?
and please add protection in routing

**Comment by bailee-crandley** [comment]
Do you know why github is marking such large sections as added and removed in ExitModal.tsx? They look pretty similar to me, are they really that different?

**Comment by bailee-crandley** [comment]
Also, there is already an active PR for the files that you're editing here so we have to be tread carefully. Maybe we should merge that one now so that you can resolve any merge conflicts before merging this one.

File: src/App.tsx (hunk 1 above)
Comment (bailee-crandley) [inline src/App.tsx:61]: By definition, a modal pops up on an existing page like the home page, it shouldn't be going to it's own page.

**Comment by aphamdd** [comment]
Hey! Yeah, I agree that the other PR should be done first, and I'll just implement the database storage once that PR is finished and close this one!

**Pull request closed without merging**